docs/shared/npm-tutorial/package-based.md

```diff
@@ -61,27 +58,37 @@ export const isEven = (x: number) => x % 2 === 0;
 {% /tab %}
 {% /tabs %}
 
-Update the `.gitignore` file to make sure you have a `dist` entry (without a leading slash) to ignore the `dist` folder inside the `is-even` package.
+Next install TypeScript (notice we're using `tsc` for the `build` script in `package.json` above). While we could install TypeScript at the package-level, it is more convenient to have it globally for the entire monorepo. Run the following command at the root of your workspace.
+
+```shell
+npm i typescript -D -W
+```
```

Comment:
Moved installing typescript until after the `packages/is-even/package.json` file is populated correctly.

(With workspaces turned on, was getting an error here when trying to run `npm i` with this file empty)